### PR TITLE
Labelling server queries page more clearly

### PR DESCRIPTION
**Pull request by hflynn**
Addresses https://trac.openmicroscopy.org/ome/ticket/10907 by renaming the page and adding an overview box to signpost the content more clearly.

Open to suggestions for a better alternative link if anyone has one.

Will be staged at https://www.openmicroscopy.org/site/support/omero5.2-staging/developers/Server/Queries.html once built.

### CODE REVIEW

File: omero/developers/Server/Queries.txt [hunk 1]
```diff
@@ -1,5 +1,12 @@
-Queries
-=======
+Using server queries internally
+===============================
+
+.. topic:: Overview
+
+    This page is aimed at internal server developers and does not
+    contain information of how to perform API queries. If that is the kind of
```
Comment (ghost) [inline omero/developers/Server/Queries.txt]: `for how` ?

**Comment by joshmoore** [comment]
I'd say this is definitely an improvement. This might actually be better suited in the javadoc itself, but rather than just making it disappear now, I say :thumbsup:

**Comment by hflynn** [comment]
Did you mean the content better placed in the javadoc @joshmoore ? I.e. migrate the content and delete this page in future?

**Comment by joshmoore** [comment]
@hflynn: exactly.

**Pull request merged**